test/ConfigTest.php

```diff
@@ -106,6 +106,7 @@ public function testPassesKnownServiceConfigKeysToServiceManagerWithConfigMethod
         $services = $this->prophesize(ServiceManager::class);
         $services->configure($expected)->willReturn('CALLED');
 
+        /** @psalm-suppress InvalidArgument */
```

Comment:
Let's add a comment here - ideally, test should be adapted, but it's fine to show that the system keeps working as it did before

Reply:
I've added a comment. I already forgot why I've added this initially TBH.